assopy/models.py

```diff
@@ -349,7 +349,7 @@ class Coupon(models.Model):
     description = models.CharField(max_length=100, blank=True)
     value = models.CharField(max_length=8, help_text='importo, eg: 10, 15%, 8.5')
 
-    user = models.ForeignKey(User, null=True, blank=True)
+    user = models.ForeignKey(User, null=True, blank=True, on_delete=models.CASCADE)
```

Comment:
is that using the same 'User' as `get_user_model()` in some of the changes above?

Reply:
No, this is actually the `assopy.User` defined earlier in the file. Renaming that model to AssopyUser was definitely a good idea.